de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/DatabaseConfiguration.java

```diff
@@ -39,6 +41,20 @@ public DatabaseConfiguration(String host, String database, String user, String p
         this.user = user;
         this.password = password;
         this.language = language;
+
+
+        // static mysql usecase - default by revision 49
+        this.setDatabaseDriver("com.mysql.jdbc.Driver");
+        this.setJdbcURL("jdbc:mysql://" + host + "/" + database);
```

Comment:
would the mysql prefix here work with the embedded db ?

Reply:
You're correct. This is a "special" constructor atm, as this is used by production code fragments whilst the other (new) constructor is only here for making HSQL embedded setup work. I'll change this with the next commits to come.